cassdegrees/ui/urls.py

```diff
@@ -52,4 +53,6 @@
     path(staff_url_prefix + 'view/subplan/', view_),
     path(staff_url_prefix + 'view/course/', view_),
     path(staff_url_prefix + 'pdf/program/', view_program_pdf),
+    path(staff_url_prefix + 'create/list/', create_list),
```

Comment:
Can you sort this into the other categories?

Reply:
Reordered listing.